src/application/worker/Interactor.js

```diff
@@ -0,0 +1 @@
+//Interactor goes here
```

Comment:
worker directory goes away, or is replaced by the persona that invokes the interactor. I would probably vote to pitch it, and we would insert a persona directory of there are so many interactors that we need to group them somehow.